### ISSUE TITLE
compatibility with other osc scripts?

### ISSUE THREAD

**Issue by asianmusicguy** [stax76/mpv.net#371]
@stax76  is there a  effective why to make mpv.net  play nice with this osc well retaking all functions 
https://github.com/cyl0/MordenX/issues/6

**Comment by ThomasEricB** [comment]
I have to agree with OP. OSC scripts in the current version works in a very unstable manner. Thumbnail Script, MordenX and tethys (https://github.com/Zren/mpv-osc-tethys) require multiple launches so they can properly work.

@stax76 Please, take a look at this. Plus, consider adding Thumbnail support into Mpv.Net

**Comment by stax76** [comment]
I made a request here:

https://github.com/cyl0/MordenX/issues/20

**Comment by stax76** [comment]
Thumbnail script is unmaintained.

mpv-osc-tethys works fine here, couldn't find a problem, having both mpv and also ffmpeg in PATH.

**Comment by stax76** [comment]
@Zren There is a severe bug happening with mpv-osc-tethys clicking on the progress bar.

**Comment by Zren** [comment]
> There is a severe bug happening with mpv-osc-tethys clicking on the progress bar.

Was wondering what it was. Seems like mpv.net has a hardcoded value in `IsMouseInOSC()` that assumed the bottom 15% of the screen is the osc. Since Tethys has a slightly taller GUI, it was changed to the bottom 20%.

https://github.com/stax76/mpv.net/commit/d6d31d8ae9df5691ac35d79875851b4c13cd8004
